src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -169,10 +170,10 @@ public static void WebSocket_Http_Duplex_BinaryStreamed()
         }
     }
 
-    [Fact]
+    [ConditionalFact(nameof(WebSockets_Supported), nameof(Root_Certificate_Installed))]
     [OuterLoop]
 #if !FEATURE_NETNATIVE
-    [ActiveIssue(420, PlatformID.AnyUnix)]
+    //[ActiveIssue(420, PlatformID.AnyUnix)]
```

Comment:
The ActiveIssue 420 applies to AnyUnix only. Wouldn't the ConditionalFact WebSockets_Support will disable this test everywhere?

There are many other similar cases like this in this PR. I will not point them out one by one, but I think you get the idea.